templates/block/block--subsitebanner-homepage.html.twig

```diff
@@ -32,16 +32,17 @@
     'block-' ~ plugin_id|clean_class,
   ]
 %}
-<div class="jumbotron jumbotron-fluid banner-full home-1" style="background-image: url({{ banner_img_url|render|striptags|trim }})">
+<div class="jumbotron jumbotron-fluid banner-full home-1" style="background-image: url({{ banner_img_url|render|striptags|trim }})" role="search" aria-labelledby="welcome-title">
   <div class="banner-full-tint h-100">
     <div class="container-fluid site-max h-100">
       <div class="row align-items-center h-100">
         <div class="col-12">
           {% block content %}
             {{ content }}
           {% endblock %}
-          <form class="form-inline justify-content-center mb-4">
-            <input class="form-control col-sm-6 col-10" type="search" placeholder="Search croydon.gov.uk" onfocus="this.placeholder = ''" onblur="this.placeholder = 'Search croydon.gov.uk'" aria-label="Search">
+          <form action="/search/google" method="get" id="search-block-form-banner" accept-charset="UTF-8" data-drupal-form-fields="edit-keys-banner" class="form-inline justify-content-center">
+          <label for="edit-keys-banner" class="visually-hidden">Search</label>
+          <input {{ attributes.addClass('form-control col-sm-6 col-8') }} type="search" placeholder="Search croydon.gov.uk" onfocus="this.placeholder = ''" onblur="this.placeholder = 'Search croydon.gov.uk'" aria-label="Search" data-drupal-selector="edit-keys-banner" name="keys">
```

Comment:
The indentation isn't looking right to me :(

Reply:
done